### PR TITLE
updated readme and singleturn files

### PR DESCRIPTION
**Pull request by shresh02**

### CODE REVIEW

**Comment by shresh02** [comment]
@microsoft-github-policy-service agree [company=""]

File: datasets/singleturn/target_world_states/builder-data/actionHit/.DS_Store
Comment (tupini07) [inline datasets/singleturn/target_world_states/builder-data/actionHit/.DS_Store:1]: minor, but I see you've committed some `.DS_Store` files. As far as I understand these are used by MacOS, specifically by finder, to save view metadata about the folders. If this was not on purpose then perhaps we could add them to our `gitignore` and remove them from git

Comment (shresh02) [reply datasets/singleturn/target_world_states/builder-data/actionHit/.DS_Store:1]: Thanks. I have added the file to .gitignore.

**Review by tupini07 — APPROVED** [review]

**Comment by shresh02** [comment]
@microsoft-github-policy-service agree

**Pull request merged**